sample/src/main/java/io/ona/kujaku/sample/activities/CardActivity.java

```diff
@@ -0,0 +1,106 @@
+package io.ona.kujaku.sample.activities;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+import com.mapbox.mapboxsdk.Mapbox;
+
+import org.json.JSONObject;
+
+import io.ona.kujaku.callbacks.AddPointCallback;
+import io.ona.kujaku.sample.BuildConfig;
+import io.ona.kujaku.sample.R;
+import io.ona.kujaku.views.KujakuMapView;
+
+public class CardActivity extends BaseNavigationDrawerActivity {
+
+    private static final String TAG = LowLevelLocationAddPointMapView.class.getName();
```

Comment:
Fix the TAG variable or remove it altogether, it is unused

Reply:
Removed.